app/src/main/java/net/skyscanner/backpack/demo/ui/ComponentListScreen.kt

```diff
@@ -19,37 +19,49 @@
 package net.skyscanner.backpack.demo.ui
 
 import android.content.Intent
+import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.items
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.stringResource
+import com.ramcosta.composedestinations.annotation.Destination
+import com.ramcosta.composedestinations.annotation.RootNavGraph
+import com.ramcosta.composedestinations.navigation.DestinationsNavigator
+import com.ramcosta.composedestinations.navigation.EmptyDestinationsNavigator
+import net.skyscanner.backpack.compose.badge.BpkBadgeType
 import net.skyscanner.backpack.compose.icon.BpkIcon
 import net.skyscanner.backpack.compose.navigationbar.BpkTopNavBar
 import net.skyscanner.backpack.compose.navigationbar.IconAction
 import net.skyscanner.backpack.compose.navigationbar.NavIcon
 import net.skyscanner.backpack.compose.navigationbar.nestedScroll
 import net.skyscanner.backpack.compose.navigationbar.rememberTopAppBarState
+import net.skyscanner.backpack.compose.theme.BpkTheme
 import net.skyscanner.backpack.compose.tokens.Settings
 import net.skyscanner.backpack.demo.R
 import net.skyscanner.backpack.demo.SettingsActivity
-import net.skyscanner.backpack.demo.compose.ComponentItem
-import net.skyscanner.backpack.demo.compose.ComponentsTitle
-import net.skyscanner.backpack.demo.meta.Component
 import net.skyscanner.backpack.demo.meta.StoriesRepository
+import net.skyscanner.backpack.demo.ui.destinations.ComponentScreenDestination
 
 @Composable
+@Destination("/")
+@RootNavGraph(start = true)
 fun ComponentListScreen(
   modifier: Modifier = Modifier,
   repository: StoriesRepository = StoriesRepository.getInstance(),
-  onClick: (Component) -> Unit,
+  navigator: DestinationsNavigator = EmptyDestinationsNavigator,
 ) {
 
   val state = rememberTopAppBarState()
 
-  Column(modifier = modifier.nestedScroll(state)) {
+  Column(modifier = modifier
+    .background(BpkTheme.colors.canvas)
```

Comment:
We need the background and full size for navigation animations to work as expected